### PR TITLE
Enable the back arrow in requests coming from Onboarding (Import, Create)

### PR DESCRIPTION
**Pull request by nibhar**

### CODE REVIEW

File: src/request/create/Create.js [hunk 1]
```diff
@@ -36,9 +36,6 @@ class Create {
         /** @type {HTMLFormElement} */
         this.$walletIdentifier = (document.querySelector('.wallet-identifier'));
 
-        /** @type {HTMLLinkElement} */
-        this.$setPasswordBackButton = (this.$setPasswordPage.querySelector('a.page-header-back-button'));
```
Comment (sisou) [inline src/request/create/Create.js]: Unused?

File: src/request/create/index.html [hunk 2]
```diff
@@ -80,9 +80,6 @@
         <div id="choose-identicon" class="page nq-card">
             <div class="page-header nq-card-header">
                 <div class="progress-indicator"></div>
-                <a tabindex="0" class="page-header-back-button">
-                    <svg class="nq-icon"><use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-arrow-left"/></svg>
-                </a>
                 <h1 data-i18n="create-heading-choose-identicon" class="nq-h1">Choose an Avatar</h1>
```
Comment (sisou) [inline src/request/create/index.html]: But what if the request comes from the onboarding menu? Then a back button would be valid.

File: src/request/create/Create.js (hunk 1 above)
Comment (nibhar) [reply src/request/create/Create.js]: yes

File: src/request/create/index.html (hunk 2 above)
Comment (nibhar) [reply src/request/create/index.html]: That is true. We do not have a back arrow for login requests, too.

**Comment by nibhar** [comment]
Might as well come from LOGIN in case of CREATE or vice versa. So a more generally applicable name was chosen. 
Feel free to recommend a renaming.

**Comment by sisou** [comment]
Hm, you are right. Ok, lets go with as it is now.

**Pull request merged**